docs/environment-setup.md

```diff
@@ -28,8 +28,17 @@ redis-server
 ```
 
 #### Windows:
+There are two methods to install Redis on Windows. We strongely recommend first approach. 
 
-We are using [Chocolatey package manager](https://chocolatey.org/) to install Redis on Windows. To get Chocolatey, simply follow this [guide](https://chocolatey.org/install) and run the following commands:
+##### Option 1: Using [WSL2 (Windows Subsystem Linux)](https://docs.microsoft.com/en-us/windows/wsl/about)   
+1. Follow Microsoft WSL2 [installation guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to complete the installation
+2. Follow Redis [installation guide](https://redis.io/topics/quickstart#installing-redis) to install Redis
+3. Suggest to install [Windows Terminal](https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-windows-terminal-optional) ,
```

Comment:
Remove extra space before comma

Reply:
Thanks for pointing that out. It has been corrected.